lucille-core/src/main/java/com/kmwllc/lucille/indexer/SolrDocRequests.java

```diff
@@ -0,0 +1,62 @@
+package com.kmwllc.lucille.indexer;
+
+import com.kmwllc.lucille.core.Document;
+import org.apache.solr.common.SolrInputDocument;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class SolrDocRequests {
+  private final List<SolrInputDocument> solrAddUpdateDocs;
+  private final Set<String> solrAddUpdateIds;
+  private final Set<String> solrDeleteIds;
```

Comment:
For the names of solrAddUpdateDocs, solrAddUpdateIds, and solrDeleteIds, I'd suggest that "solr" isn't needed because it's already part of the class name "SolrDocRequests". However, addUpdateDocs, addUpdateIds, and deleteIds sound like they could be method names, so I'm suggesting "docsToAddOrUpdate", "idsToAddOrUpdate", and "idsToDelete"